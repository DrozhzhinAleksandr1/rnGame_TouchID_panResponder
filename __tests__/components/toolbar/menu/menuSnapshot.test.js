import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import { Menu } from '/components/toolbar/menu/menu';

it('create snapshot toolbar', () => {
    const renderer = new ShallowRenderer()
    let toolbarSnapshot = renderer.render(<Menu />);
    expect(toolbarSnapshot).toMatchSnapshot();
});