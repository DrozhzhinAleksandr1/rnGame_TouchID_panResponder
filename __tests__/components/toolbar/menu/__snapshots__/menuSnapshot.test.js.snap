// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create snapshot toolbar 1`] = `
<TouchableWithoutFeedback
  accessibilityLabel="ButtonMenuID"
  onPress={[Function]}
  testID="ButtonMenuID"
>
  <Component
    style={
      Object {
        "alignItems": "center",
        "height": "100%",
        "justifyContent": "center",
        "width": 56,
      }
    }
  >
    <Icon_menu />
  </Component>
</TouchableWithoutFeedback>
`;
